packages/@react-native-windows/cli/src/config/projectConfig.ts

```diff
@@ -195,6 +197,13 @@ export function projectConfigWindows(
       sourceDir,
       path.join(sourceDir, result.solutionFile),
     );
+
+    const experimentalFeatures = configUtils.getExperimentalFeatures(
+      path.dirname(path.join(sourceDir, result.solutionFile)),
+    );
+    if (experimentalFeatures) {
+      result.experimentalFeatures = experimentalFeatures;
```

Comment:
I could not find current usage yet, is that intentional?

Reply:
yeah, this is setup for the telemetry refactor, i just thought it was better as it's own PR since it affects native builds and the telemetery work is all js

Reply:
also, this can and should be backported to 65, whereas the telemetry refactor itself might not be